tdrs-backend/tdpservice/parsers/case_consistency_validator.py

```diff
@@ -24,25 +25,41 @@ def __init__(self, header, program_type, stt_type, generate_error):
         self.current_case_hash = None
         self.case_has_errors = False
         self.section = header["type"]
-        self.case_is_section_one_or_two = self.section in {'A', 'C'}
+        self.case_is_section_one_or_two = self.section in {"A", "C"}
         self.program_type = program_type
+        self.is_ssp = self.program_type == "SSP"
```

Comment:
@elipe17 what is this line doing?

Reply:
This lets the validator know if it is validating an SSP file. It was made a member out of convenience since it needed to be used across a few member functions to support these changes. Originally it was a local variable to only one member method.